src/main/python/deep_qa/layers/noisy_or.py

```diff
@@ -0,0 +1,88 @@
+from keras import backend as K
+from keras.layers import Layer
+from keras.initializations import identity
+from keras.constraints import Constraint, NonNeg
+from keras.regularizers import L1L2Regularizer
+from overrides import overrides
+
+class BetweenZeroAndOne(Constraint):
+    """Constrains the weights to be between zero and one
+    """
+    def __call__(self, p):
```

Comment:
Oh, yes, definitely.  I forgot that you were overriding a method from keras.  If you're overriding a method, you need to match its signature exactly, so you did the right thing.  It's keras' fault that the code is less readable than it should be; there's not much we can do about that.